src/main/resources/.index_summary_embedding_index_mapping.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
What's this new index for?

Reply:
this index served as vector store for index summary embeddings.